src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +30,14 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        if (nextSequenceNumber == SEQUENCE_NUMBER_UNINITIALISED_INDEX) {
```

Comment:
I see.
I forgot to change the Parser method, will change it after finishing T5A4!